spec/features/editing_images/upload_lead_image_asset_manager_down_spec.rb

```diff
@@ -33,6 +33,6 @@ def then_i_should_see_an_error
   end
 
   def and_the_image_does_not_exist
-    expect(page).to have_content(I18n.t("document_images.index.no_existing_image"))
+    expect(page).to_not have_content(I18n.t("document_images.index.existing_image"))
```

Comment:
Sorry for being vague. It’s this line here. It’s very easy for a page to not have content - maybe we change the workflow so it’s never there anyway. Do you think there’s a positive way to check there’s no image?

Reply:
Ah okay, understood!

Reply:
Hmmm well you could check that there is an image tag with the css class but this may be flakey due to changes in the styling.

Reply:
Ah ah, just found that content-publisher has a table `images` which keeps a record of each image uploaded. So could check there to make sure nothing has been created!

Reply:
I'll merge this then make the tests better in another pr